package.json

```
@@ -20,12 +20,19 @@
     "react-social-login-buttons": "^3.1.0"
   },
   "devDependencies": {
+<<<<<<< HEAD
+    "@types/node": "^12.19.11",
+    "@types/react": "^16.14.2",
+    "@types/react-dom": "^16.9.10",
+    "typescript": "^4.1.3"
+=======
     "@types/next-auth": "^3.1.18",
     "@types/node": "^12.12.21",
     "@types/react": "^16.9.16",
     "@types/react-dom": "^16.9.4",
     "cross-env": "^7.0.3",
     "typescript": "^4.0"
+>>>>>>> dev
```

This needs to be merged
```suggestion
    "@types/next-auth": "^3.1.18",
    "@types/node": "^12.12.21",
    "@types/react": "^16.9.16",
    "@types/react-dom": "^16.9.4",
    "cross-env": "^7.0.3",
    "typescript": "^4.0"
```